src/jit/compiler.h

```diff
@@ -7352,39 +7352,39 @@ XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
     XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
     */
 
-    // Get highest available instruction set for floating point codegen
-    InstructionSet getFloatingPointInstructionSet()
+    // Get highest available level for floating point codegen
+    SIMD_Level getFloatingPointCodegenLevel()
```

Comment:
I think we should change this to `getSIMDSupportLevel()`, and avoid having one call the other. It seems that `getFloatingPointInstructionSet()` is currently only used in a SIMD context, and now that the return value has SIMD in its name, and is defined in simd.h, it makes more sense.

Reply:
Yes, I also detected that `getFloatingPointInstructionSet` never used to directly control floating-point codegen. Can I leave it here and refactor the codgen strategy in next PRs?